DelvUI/Interface/Party/PartyFramesBar.cs

```diff
@@ -58,6 +59,7 @@ PartyFramesRaiseTrackerConfig raiseTrackerConfig
             _raiseTrackerConfig = raiseTrackerConfig;
 
             _nameLabelHud = new LabelHud(id + "_nameLabel", config.NameLabelConfig);
+            _healthLabelHud = new LabelHud(id + "_nameLabel", config.HealthLabelConfig);
```

Comment:
Why not call it _healthLabel ?

Reply:
Sorry, I copied the above and didn't change it.